src/pages/NewChatPage.tsx

```diff
@@ -241,6 +244,11 @@ function NewChatPage({betas, isGroupChat, personalDetails, reports, isSearchingF
         };
     }, []);
 
+    useEffect(() => {
+        setSelectedOptions(getGroupParticipants());
+        // eslint-disable-next-line react-hooks/exhaustive-deps -- Overwrite participants only if the draft changes
+    }, [newGroupDraft?.participants]);
```

Comment:
This may raise the question on why we don't always use the draft participants i.e. `selectedOptions = newGroupDraft.participants`. The reason is that the draft participants are set only after the user presses the next button.

Reply:
Great point. Makes me wonder why we don't start saving them when you first tap "Add to group". Feels like we should have designed it that way (fixing that seems outside the scope of your current changes). 🤔